tests/pyspark/wranglers/test_interval_identifier.py

```diff
@@ -233,3 +233,25 @@ def test_no_groupby_order_columns(test_case, wrangler, groupby_order,
     else:
         test_output = wrangler_instance.fit_transform(test_input)
         assert_pyspark_pandas_equality(test_output, expected)
+
+
```

Comment:
This test should be run against all available wrangler implementations.
`@pytest.mark.parametrize(**WRANGLER_KWARGS)`